packages/nds-card/__tests__/__snapshots__/nds-card.test.js.snap

```diff
@@ -16,18 +16,6 @@ exports[`Card should match snapshot with some default attributes 1`] = `
           "elementType": "a",
         }
       }
-      metadata={
```

Comment:
Why is this metadata removed from the snapshot when that test setup hasn't changed?